packages/sdk/contractkit/src/base.ts

```diff
@@ -6,6 +6,7 @@ export enum CeloContract {
   DowntimeSlasher = 'DowntimeSlasher',
   Election = 'Election',
   EpochRewards = 'EpochRewards',
+  ERC20 = 'ERC20',
```

Comment:
this seems like the wrong place for this to live

Reply:
is "ERC20" a `CeloContract`?

Reply:
We should define what we think it is a CeloContract. In a way, we think of those as the contracts supported/maintained by celo. The ERC20 is a standard accepted by everyone.
Let's say that the contract does not exists in the ethereum network, we would probably want to handle and support that contract.
So, in a way, it is something standard for celo too, we are just taking the advantage of reuse it from other network.

If we think of the CeloContracts as only the ones that live on the registry, we should also remove the multisig from that list.

If we think of the CeloContracts, as the one accepted by the community, all the ercXXX accepted, should be supported too